src/RemoteConfig.php

```diff
@@ -112,10 +117,43 @@ public function setCache(CacheInterface $cache)
         $this->cache = $cache;
     }
 
+    public function updateCacheData(string $client, array $data, bool $canExpire = true)
+    {
+        $uri = $this->buildUri($this->application, $client, $this->environment);
+        $cacheKey = $this->buildCacheKey($uri);
+
+        $cache = $this->getCache();
+        if (method_exists($cache, 'tags')) {
+            $cache = $cache->tags($this->getCacheTags($client));
+        }
+
+        $cacheLifeTime = $canExpire ? $this->cacheLifeTime : null;
```

Comment:
`null` value change behavior of method "set"? If it is the same of `false` value, prefer to use same type (bool) to default value. What do u think?